01-Plot-Raster.Rmd

```diff
@@ -113,67 +126,83 @@ plot(DSM_HARV,
 ```
 
 # Plotting Data Using Breaks
-
 Often we may want to view our data subsetted by a range of values rather than
-using a continuous color ramp. This is comparable to a "classified" map in a tool
-like ArcGIS or QGIS.
-
-To begin we want use the histogram of our data to determine where the breaks in 
-our data should go. When can use `breaks=` to tell `R` to use fewer or more breaks
-or bins in our histogram. More on that below:
+using a continuous color ramp. This is comparable to a "classified" map in a
+tool like ArcGIS or QGIS.
 
-> R Bloggers Histogram Overview: <a href="http://www.r-bloggers.com/basics-of-histograms/" target="_blank">More 
-on histograms in R</a>
+Unless there is a reason for having specific breaks, we want to use a
```

Comment:
This is good but a bit confusing although i know what you are trying to say.. How about

We can use a histogram to observe the distribution of values in our data and determine where breaks should go.